instaclustr/resource_cluster.go

```diff
@@ -736,7 +1068,7 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
-func getBundlesFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
+func getBaseBundlesFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
```

Comment:
[question] I don't quite know what a base bundle is? are you talking about root bundle, primary bundle, other bundle?

Reply:
oh I realised this isn't just your change calling them a base bundle. I don't know if this is a term we commonly use?